src/pages/ReferralDetailsPage.tsx

```diff
@@ -14,49 +14,38 @@ import useSingleExecution from '@hooks/useSingleExecution';
 import useTheme from '@hooks/useTheme';
 import useThemeStyles from '@hooks/useThemeStyles';
 import Clipboard from '@libs/Clipboard';
+import type {ReferralDetailsNavigatorParamList} from '@libs/Navigation/types';
 import * as Link from '@userActions/Link';
 import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
 import SCREENS from '@src/SCREENS';
+import type {Account} from '@src/types/onyx';
 import * as ReportActionContextMenu from './home/report/ContextMenu/ReportActionContextMenu';
 
-const propTypes = {
-    /** Navigation route context info provided by react navigation */
-    route: PropTypes.shape({
-        params: PropTypes.shape({
-            /** The type of the content from where CTA was called */
-            contentType: PropTypes.string,
-        }),
-    }).isRequired,
-
+type ReferralDetailsPageOnyxProps = {
     /** The details about the account that the user is signing in with */
-    account: PropTypes.shape({
-        /** The primaryLogin associated with the account */
-        primaryLogin: PropTypes.string,
-    }),
+    account: OnyxEntry<Account>;
 };
 
-const defaultProps = {
-    account: null,
-};
+type ReferralDetailsPageProps = ReferralDetailsPageOnyxProps & StackScreenProps<ReferralDetailsNavigatorParamList, typeof SCREENS.REFERRAL_DETAILS>;
 
-function ReferralDetailsPage({route, account}) {
+function ReferralDetailsPage({route, account = null}: ReferralDetailsPageProps) {
```

Comment:
I think we don't need `null` as a default value here
```suggestion
function ReferralDetailsPage({route, account}: ReferralDetailsPageProps) {
```

Reply:
Done!